tests/packages/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "packages-test",
+  "version": "1.0.0",
+  "description": "Test module resolution of asc",
+  "main": "index.js",
+  "scripts": {
+    "test": "node exec.js run build"
+  },
+  "keywords": [
+    "test",
+    "assemblyscript"
+  ],
+  "author": "Willem Wyndham",
+  "license": "MIT",
```

Comment:
The package files still seem too verbose. Like, the license should be that of the compiler (right?) and there's not really a need for description, author and maybe other fields I think.

Reply:
I removed all of the the other package.json files; losing some info for the IDE, but cleans it up a lot).  Also changed the license.